instagram-app/src/components/PostContainer/PostsPage.jsx

```diff
@@ -0,0 +1,107 @@
+import '../../App.css';
+import React, { Component } from 'react'
+import Navigation from '..//SearchBar/SearchBar'
+import data from '../../dummy-data'
+import Posts from '../PostContainer/PostContainer'
+import {IconContext} from 'react-icons'
+import $ from 'jquery'
+import 'bootstrap/js/dist/modal'
+import CommentModal from '../Modals/CommentModal'
+import Fuzzy from 'fuzzy'
+
+
+export default class PostsPage extends Component {
+    constructor(props){
+        super(props)
+        this.state={
+          searching:'',
+          data:[],
+          user:'AshenPheonix',
+          newComment:'',

```

Comment:
Don't forget to clean up after working on something. You wouldn't want things like this that are easy to forget making it into production.